docs/source/indepth_tutorial/directed-search.ipynb

```diff
@@ -98,9 +98,9 @@
      "name": "stderr",
```

Comment:
<p>What does this do? And should it give an output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>It gives an output: the dimensions of the new archive. The key point is to show that the merged archive contains fewer solutions that the sum of solutions over all archives. this is brought out a bit more clearly in the text now.</p>

Reply:
<p>Nice!</p>